specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Enable_api_compatibility_attribute_checks.cs

```diff
@@ -0,0 +1,72 @@
+namespace Rules.MS_Build.Enable_api_compatibility_attribute_checks;
+
+public class Reports
+{
+    [Test]
+    public void on_missing_property() => new EnableApiCompatibilityAttributeChecks()
+        .ForInlineCsproj(@"
+            <Project Sdk=""Microsoft.NET.Sdk"">
+
+              <PropertyGroup>
+                <TargetFramework>net8.0</TargetFramework>
+                <EnablePackageValidation>true</EnablePackageValidation>
+              </PropertyGroup>
+
+            </Project>
+        ")
+        .HasIssues(Issue.WRN("Proj0251", "Define the <ApiCompatEnableRuleAttributesMustMatch> node with value 'true' or remove the <EnablePackageValidation> node with value 'true'.").WithSpan(00, 00, 07, 22));
+
+    [Test]
+    public void on_disabled_property() => new EnableApiCompatibilityAttributeChecks()
+        .ForInlineCsproj(@"
+            <Project Sdk=""Microsoft.NET.Sdk"">
+
+              <PropertyGroup>
+                <TargetFramework>net8.0</TargetFramework>
+                <EnablePackageValidation>true</EnablePackageValidation>
+                <ApiCompatEnableRuleAttributesMustMatch>false</ApiCompatEnableRuleAttributesMustMatch>
+              </PropertyGroup>
+
+            </Project>
+        ")
+        .HasIssues(Issue.WRN("Proj0251", "Define the <ApiCompatEnableRuleAttributesMustMatch> node with value 'true' or remove the <EnablePackageValidation> node with value 'true'.").WithSpan(05, 16, 05, 102));
```

Comment:
Should this rule also mention "remove the ApiCompatEnableRuleAttributesMustMatch node if you are removing the EnablePackageValidation node"? Does it make sense to keep the first if the second is not there?